src/pages/api/client_sessions/create.ts

```diff
@@ -31,7 +31,11 @@ export default withRouteSpec({
     (req.headers["user-identifier-key"] as string | undefined) ??
     (req.headers["seam-user-identifier-key"] as string | undefined)
 
-  if (req.auth.auth_mode === "publishable_key" && user_identifier_key == null) {
+  if (
+    req.auth.type === "client_session" &&
```

Comment:
Client sessions cannot create more client sessions, this check only applies to `publishable_key` auth.

Reply:
Based on the implementation in seam-connect only `client_sessions` have the method property in the auth middleware with `publishable_key` as an option, is this incorrect?

Reply:
The endpoint should always return 403 if client session auth.